sdk/monitor/monitor-opentelemetry-exporter/test/unit/utils/eventhub.test.ts

```diff
@@ -53,11 +53,13 @@ describe("#parseEventHubSpan(...)", () => {
     span.setAttributes(attributes);
 
     parseEventHubSpan(span, envelope);
-    assert.strictEqual(envelope.data?.baseData?.type, attributes[AzNamespace]);
-    assert.strictEqual(envelope.data?.baseData?.target, `${peerAddress}/${destination}`);
 
-    assert.strictEqual(envelope.data?.baseData?.source, undefined);
-    assert.strictEqual(envelope.data?.baseData?.measurements, undefined);
+    const baseData = envelope.data?.baseData as RemoteDependencyData;
```

Comment:
how do you imagine folks actually using this? I submit if they have to do their own casts, we're not providing much value to TS.

Reply:
Agreed. The `baseData` types won't be very useful until we can use some `oneOf` schemas in the swagger file